packages/blockchain-wallet-v4-frontend/src/scenes/Settings/General/LinkedBanks/template.success.tsx

```diff
@@ -33,13 +31,30 @@ const StyledSettingsContainer = styled(SettingContainer)`
 `
 
 const Success: React.FC<InjectedFormProps<{}, Props> & Props> = (props) => {
+  const { data: paymentMethods, isLoading: isLoadingPaymentMethods } = useRemote(
+    selectors.components.buySell.getBSPaymentMethods
+  )
+
+  const bankLimit = useMemo(() => {
+    if (isLoadingPaymentMethods) return
+
+    return (
+      paymentMethods?.methods.find((method) => method.type === BSPaymentTypes.BANK_TRANSFER)
```

Comment:
Would it make more sense to assign this to a variable in the function and not returning it? At a glance this looks like a ternary, not that there is a default value being returned if no ´.limits´ is present or no `paymentMethods.methods` matching the find are present.
Also, why are these the default limits? I mean, this must be specified somewhere but it does look like magic numbers, it being "hidden" here.
Doing something like `return (bankPaymentMethod?.limits || DEFAULT_LIMIT)` can be a bit more expressive..

Reply:
Agree magic numbers should not be hidden in the code

Reply:
actually, if the api is not providing any limits it should not show any limits at all, correct?

Reply:
> Would it make more sense to assign this to a variable in the function and not returning it? At a glance this looks like a ternary, not that there is a default value being returned if no `.limits` is present or no `paymentMethods.methods` matching the find are present.
> 
> Also, why are these the default limits? I mean, this must be specified somewhere but it does look like magic numbers, it being "hidden" here.
> 
> Doing something like `return (bankPaymentMethod?.limits || DEFAULT_LIMIT)` can be a bit more expressive..